libr/bin/bfile.c

```diff
@@ -669,7 +670,7 @@ R_API RBinFile *r_bin_file_find_by_name(RBin *bin, const char *name) {
 	return bf;
 }
 
-R_API RBinFile *r_bin_file_find_by_name_n(RBin *bin, const char *name, int idx) {
+RBinFile *r_bin_file_find_by_name_n(RBin *bin, const char *name, int idx) {
```

Comment:
We have R_IPI for the internal apis

Reply:
Mmm I see R_IPI is re-defined in some files... is it safe to use?